apps/src/javalab/JavalabEditor.jsx

```diff
@@ -360,6 +362,40 @@ class JavalabEditor extends React.Component {
     });
   }
 
+  // This moves the active tab to the left in the tab menu
+  moveTabLeft() {
+    const {activeTabKey, orderedTabKeys} = this.props;
+    let newTabs = [...orderedTabKeys];
+    let index = newTabs.indexOf(activeTabKey);
+    if (index > 0) {
+      this.swapTabsSetKeyCloseTab(index - 1, index, newTabs);
+    }
+  }
+
+  // This moves the active tab to the right in the tab menu
+  moveTabRight() {
+    const {activeTabKey, orderedTabKeys} = this.props;
+    let newTabs = [...orderedTabKeys];
+    let index = newTabs.indexOf(activeTabKey);
+    if (index < newTabs.length - 1) {
+      this.swapTabsSetKeyCloseTab(index, index + 1, newTabs);
+    }
+  }
+
+  swapTabsSetKeyCloseTab(index1, index2, newTabs) {
+    const {setOrderedTabKeys} = this.props;
```

Comment:
nit: if you're only pulling `setOrderedTabKeys` out of props, it's probably simpler to just call `this.props.setOrderedTabKeys(...)` directly.

Reply:
Sounds good - will do!